grpc/service_config/service_config.proto

```diff
@@ -36,21 +36,32 @@ option java_outer_classname = "ServiceConfigProto";
 
 // Configuration for a method.
 message MethodConfig {
-  // The names of the methods to which this configuration applies. There must
-  // be at least one name. Each name entry must be unique across the entire
-  // ClientConfig. If the 'method' field is empty, then this MethodConfig
-  // specifies the defaults for all methods for the specified service.
+  // The names of the methods to which this configuration applies.
+  // - MethodConfig without names (empty list) will be skipped.
+  // - Each name entry must be unique across the entire ClientConfig.
```

Comment:
s/ClientConfig/ServiceConfig/

(pre-existing problem, but please fix while you're in here)

Reply:
Done